lib/sentry/default_event_filter.ex

```diff
@@ -3,7 +3,15 @@ defmodule Sentry.DefaultEventFilter do
 
   @moduledoc false
 
-  def exclude_exception?(%x{}, :plug) when x in [Phoenix.Router.NoRouteError] do
+  @ignored_exceptions [
+    Phoenix.Router.NoRouteError,
+    Plug.Parsers.RequestTooLarge,
```

Comment:
I think this should be `RequestTooLargeError`

Reply:
Good catch!